sensors/weather/sensorhub-driver-storm/build.gradle

```diff
@@ -5,13 +5,25 @@ version = '0.4.0'
 dependencies {
   implementation 'org.sensorhub:sensorhub-core:' + oshCoreVersion
   implementation project(':sensorhub-utils-grid')
+  // UCAR NetCDF dependencies
+  embeddedImpl 'edu.ucar:cdm:4.6.9'
+
 }
 
 // add info to OSGi manifest
 osgi {
   manifest {
     attributes('Bundle-Vendor': 'Botts Innovative Research Inc.')
     attributes('Bundle-Activator': 'org.sensorhub.impl.sensor.mrms.Activator')
+    attributes('Import-Package': '''
+      !com.amazonaws.*, !com.sun.*, !commonj.work.*, !java.*, !javax.crypto.*, !javax.imageio.*,
+      !javax.net.*, !javax.naming.*, !javax.management.*, !javax.security.*, !javax.sql.*, !javax.swing.*,
+      !javax.transaction.*, !javax.xml.*, !kotlin.*, !oracle.*, !org.apache.avalon.*, !org.apache.log.*,
+      !org.apache.log4j.*, !org.apache.xerces.*, !org.bouncycastle.jce.*, !org.ietf.jgss.*, !org.jaxen.*,
+      !org.jboss.*, !org.joda.*, !org.quartz.jobs.*, !org.terracotta.*, !scala.util.*, !ucar.jpeg.*,
+      !weblogic.*, !org.w3c.dom.*, !org.xml.*, !ucar.ma2.*, !ucar.nc2.*, !ucar.unidata.*,
+      org.osgi.service.repository;resolution:=optional,
```

Comment:
Don't think we need to import "org.osgi.service.repository;resolution:=optional"

Reply:
Ok for building and installing.: bff5b1598ceda162b07185739aaf0d11b18fc4c8